### PR TITLE
refactor: :recycle: reduce number of vs-code recommendations

### PR DESCRIPTION
**Pull request by tomleo**
### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [x] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Removing some vs-code recommendations for this mono-repo. Things like django and python are not used in any of the projects in this repo.

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.
- [ ] I have documented testing approach

### CODE REVIEW

File: .vscode/extensions.json [hunk 1]
```diff
@@ -1,25 +1,17 @@
...
         "octref.vetur",
         "dbaeumer.vscode-eslint",
-        "EditorConfig.EditorConfig",
+        "editorconfig.editorconfig",
         "mikestead.dotenv",
-
-        // Recommended
         "esbenp.prettier-vscode",
         "lokalise.i18n-ally",
-        "eamodio.gitlens",
```
Comment (tomleo) [inline .vscode/extensions.json:17]: This is a personal productivity tool for some, but not directly needed for this project. Personally an extension I don't like or use.

File: .vscode/extensions.json [hunk 2]
```diff
@@ -1,25 +1,17 @@
...
-        "Zignd.html-css-class-completion",
-        "reduckted.vscode-gitweblinks",
```
Comment (tomleo) [inline .vscode/extensions.json:19]: Similar to gitlens, this feels more like a personal productivity tool, than something that's necessary for this particular project.

File: .vscode/extensions.json [hunk 3]
```diff
@@ -1,25 +1,17 @@
 {
     "recommendations": [
-        // "Required"
-        "ms-python.python",
```
Comment (tomleo) [inline .vscode/extensions.json:4]: We're not using python in this project.

File: .vscode/extensions.json [hunk 4]
```diff
@@ -1,25 +1,17 @@
...
         "davidanson.vscode-markdownlint",
         "stylelint.vscode-stylelint",
         "redhat.vscode-yaml",
-        "batisteo.vscode-django",
```
Comment (tomleo) [inline .vscode/extensions.json:8]: We're not using django in this project.

**Review by RyanMulready — APPROVED** [review]
Changes look fine. FWIW I've started working on another vscode setup using Volar instead of Vetur as part of the CMS project. 

```
{
    "recommendations": [
        "Vue.volar",
        "Vue.vscode-typescript-vue-plugin",
        "davidanson.vscode-markdownlint",
        "stylelint.vscode-stylelint",
        "dbaeumer.vscode-eslint",
        "esbenp.prettier-vscode",
        "rvest.vs-code-prettier-eslint",
        "EditorConfig.EditorConfig",
        "vivaxy.vscode-conventional-commits"
    ]
}
```

**Pull request merged**